requirements.txt

```diff
@@ -1,5 +1,5 @@
 # Neuro SAN - Core dependency for agent session management
-neuro-san>=0.6.19
+neuro-san>=0.6.20
```

Comment:
Need to use the latest version of neuro-san for tweaks in agent network validation that allow for reachability tests to look at a CodedTool's args.tools declaration.